VisualChatGPTStudio.csproj

```diff
@@ -151,7 +151,7 @@
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
     <PackageReference Include="OpenAI">
-      <Version>1.5.0</Version>
+      <Version>1.6.0</Version>
```

Comment:
Set to 1.7.0

The current version is 1.6.1

Reply:
Rebased and resolved conflict